src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -737,6 +737,21 @@ class ADIOSWriter : public ILightweightPlugin
         /* define (sizes for) meta attributes */
         defineMetaAttributes(threadParams);
 
+        /* write created variable values */
```

Comment:
why did we move that up again? because of `forEachCollectFieldsSizes` and `forEachCollectFieldsSizes`?